config/helm/aws-node-termination-handler/README.md

```diff
@@ -57,7 +57,7 @@ Parameter | Description | Default
 `gracePeriod` | (DEPRECATED: Renamed to podTerminationGracePeriod) The time in seconds given to each pod to terminate gracefully. If negative, the default value specified in the pod will be used. | `30`
 `podTerminationGracePeriod` | The time in seconds given to each pod to terminate gracefully. If negative, the default value specified in the pod will be used. | `30`
 `nodeTerminationGracePeriod` | Period of time in seconds given to each NODE to terminate gracefully. Node draining will be scheduled based on this value to optimize the amount of compute time, but still safely drain the node before an event. | `120`
-`ignoreDaemonsSets` | Causes kubectl to skip daemon set managed pods | `true`
+`ignoreDaemonSets` | Causes kubectl to skip daemon set managed pods | `true`
```

Comment:
nice catch! Can you submit this in another PR?

Reply:
https://github.com/aws/aws-node-termination-handler/pull/199